yarn-project/boxes/blank-react/src/contracts/Nargo.toml

```diff
@@ -5,4 +5,4 @@ compiler_version = "0.1"
 type = "contract"
 
 [dependencies]
-aztec = { git="https://github.com/AztecProtocol/aztec-packages", tag="master", directory="yarn-project/aztec-nr/aztec" }
+aztec = { path = "../../../../aztec-nr/aztec" }
```

Comment:
I assume you did this so it points to current aztec-nr rather than what is there on master?
* If someone npm installs this blank box, does their nargo toml now expect aztec-nr in that location rather than reading from github?

Reply:
Yep, issue is that if we are changing anything in the aztec-noir that is used, then the boxes are built with the old version. So in cases where we alter the context and inputs as this pr, it will fail in CI because it is using the old ones. 

Did it this way since that will work, but yes for npm installs not sure how annoying this is. @dan-aztec

Reply:
ah so rahul's reasoning is why we pointed to github instead of local, to make it easier for people who locally just copy the subpackage by itself - we overwrite the `master` tag with the `aztec-cli --version` number to try to keep everything on the same release.

this does make it annoying to do the manual build though, maybe i should add "blank" to `noir-contracts` and then a script to recompile/copy things over

Reply:
1. If we think such PRs will be infrequent, I propose a disgusting hack where we merge the nargo.toml as is in your pr. And then immediately follow up with a PR that reverts these `nargo.toml` changes
2. If we think such PRs will be quite frequent - think of better ways now -> Perhaps first try to see if `aztec-nr` exists at `"../../../../aztec-nr/aztec"` and if not, then use the master branch 🤷

Reply:
since the `artifacts` are re-compiled now, can we revert the changes to the box `Nargo.toml` dependencies?  I'll add instructions on how to patch boxes going forward